templates/vhost_conf.erb

```diff
@@ -69,11 +70,6 @@ if @locations
 <%
     loc.sort.each do |key, values|
       if key != 'name'
-        if key == 'proxy_pass'
--%>
-    include proxy_params;
```

Comment:
Actually it has never been part of Nginx distribution..

It's osfamily-specific file. I fixed my PR.